src/math/Quaternion.d.ts

```diff
@@ -117,15 +117,10 @@ export class Quaternion {
 		dst: number[],
 		dstOffset: number,
 		src0: number[],
-		srcOffset: number,
+		srcOffset0: number,
 		src1: number[],
-		stcOffset1: number,
+		srcOffset1: number,
 		t: number
-	): Quaternion;
-
-	/**
-   * @deprecated Use {@link Vector#applyQuaternion vector.applyQuaternion( quaternion )} instead.
-   */
-	multiplyVector3( v: any ): any;
```

Comment:
Why have you removed the declaration of `multiplyVector3()`?

Reply:
It doesn't seem to exist in the implementation: https://github.com/mrdoob/three.js/blob/dev/src/math/Quaternion.js

Reply:
It's defined in `THREE.Legacy.js`. So the method is still accessible.